### PR TITLE
usi-dock: set correct chip type before update.

### PR DESCRIPTION
**Pull request by victor-cheng**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/usi-dock/fu-usi-dock-mcu-device.c [hunk 1]
```diff
@@ -707,6 +708,30 @@ fu_usi_dock_mcu_device_write_firmware(FuDevice *device,
...
+fu_usi_dock_mcu_device_prepare(FuDevice *device,
+			       FuProgress *progress,
+			       FwupdInstallFlags flags,
+			       GError **error)
+{
+	FuUsiDockMcuDevice *self = FU_USI_DOCK_MCU_DEVICE(device);
+	guint8 inbuf[] = {USBUID_ISP_DEVICE_CMD_SET_CHIP_TYPE, 1, 1};
+
+	if (fu_device_has_guid(device, USI_DOCK_NON_IOT_INSTANCE_ID) &&
+	    g_strcmp0(fu_device_get_version(device), "10.10") == 0) {
```
Comment (hughsie) [inline plugins/usi-dock/fu-usi-dock-mcu-device.c:721]: is this only one model and version combination, or could there be more in the future? I'm wondering if this is better suited as a private flag.

Comment (superm1) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:721]: That's why I was looking for a comment for more context too..

Comment (victor-cheng) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:721]: Only such a combination requires additional commands to avoid some problems after the update.

Comment (hughsie) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:721]: Okay, lets merge this, but if we need any more we can move it to a quirk.

**Review by hughsie — APPROVED** [review]

**Pull request merged**